core/src/main/java/hudson/tasks/ArtifactArchiver.java

```diff
@@ -214,6 +223,14 @@ public boolean isCaseSensitive() {
         this.caseSensitive = caseSensitive;
     }
 
+    public boolean isFollowSymlinks() {
+        return followSymlinks;
```

Comment:
How confident are we that this never breaks thanks to `readResolve`?

Reply:
I'm pretty confident. If we had another test or two, I'd be sufficiently confident.